world.js

```diff
@@ -797,6 +852,9 @@ world.addObject = (contentId, parentId = null, position = new THREE.Vector3(), q
 };
 world.removeObject = removeInstanceId => {
   state.transact(() => {
+    const index = pointers.findIndex(x => x.instanceId === removeInstanceId);
+    if (index !== undefined) pointers.splice(index, 1);
```

Comment:
This won't work because `findIndex` returns a number (check against -1 instead).

Reply:
50e067f